packages/web-config-server/src/apiV1/dataBuilders/generic/latestData/multiDataValuesLatestSurvey.js

```diff
@@ -13,7 +13,7 @@ export const multiDataValuesLatestSurvey = async (
   );
   const latestPeriod = getMostRecentPeriod(surveyDatesResponseDataValues.map(srv => srv.period));
   if (!latestPeriod) {
-    return null;
+    return { data: [] };
```

Comment:
@edmofro can I also sneak this fix in here? This is a current issue in `dev` and `prod` that the e2e tests found. It fixes the last report that consistently fails the tests because of a code error (at least with the current orgUnitMap) 🎉

This also made me think that we should probably also test reports when no data are present. One way to do so is provide an alternative orgUnitMap and also run the tests against that.

Reply:
Yes and good idea!